ios/RNOneginiSdk.swift

```diff
@@ -358,6 +358,20 @@ class RNOneginiSdk: RCTEventEmitter, ConnectorToRNBridgeProtocol {
         }
     }
 
+    @objc
+    func validatePinWithPolicy(_ pin: (String),
+                        resolver resolve: @escaping RCTPromiseResolveBlock,
+                        rejecter reject: @escaping RCTPromiseRejectBlock) -> Void {
+        userClient.validatePin(withPolicy: pin) {
+            (_ , error) -> Void in
```

Comment:
this line should be aligned with the line above:

`userClient.validatePin(withPolicy: pin) { (_ , error) -> Void in`